cc-check.go

```diff
@@ -69,7 +69,11 @@ var requiredKernelModules = map[string]kernelModule{
 	"kvm_intel": {
 		desc: "Intel KVM",
 		parameters: map[string]string{
-			"nested":             "Y",
+			"nested": "Y",
+			// "VMX Unrestricted mode support". This is used
```

Comment:
@mcastelino that's weird cause Azure VM does not have this option enabled and I have been able to get CC3.0 running with our current setup (qemu-lite with pc machine type) + `disable-modern=true` for virtio devices.